sdk/keyvault/keyvault-keys/src/lro/delete/operation.ts

```diff
@@ -36,16 +36,11 @@ export class DeleteKeyPollOperation extends KeyVaultKeyPollOperation<
    * Since the Key Vault Key won't be immediately deleted, we have {@link beginDeleteKey}.
    */
   private async deleteKey(name: string, options: DeleteKeyOptions = {}): Promise<DeletedKey> {
-    const requestOptions = operationOptionsToRequestOptionsBase(options);
-    const span = createSpan("generatedClient.deleteKey", requestOptions);
+    const { span, updatedOptions } = createSpan("generatedClient.deleteKey", options);
```

Comment:
Future TODO: these aren't really generated clients, they're still wrappers, so we might just want to come up with another prefix _or_ reuse the parent's prefix instead.